sunpy/map/sources/stereo.py

```diff
@@ -42,6 +42,10 @@ def __init__(self, data, header, **kwargs):
         # date FITS keyword
         if ('date_obs' in self.meta) and not('date-obs' in self.meta):
             self.meta['date-obs'] = self.meta['date_obs']
+        # fix CROTA to CROTAn
+        if "crota" in self.meta:
```

Comment:
```suggestion
        if "crota" in self.meta and "crota2" not in self.meta:
```
Just to be on the safe side I think it's worth checking there isn't already a crota2.